app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name, health=100, hidden=False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        a = f"Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}"
+        return f'{{{a}}}'
+
+
+class Herbivore(Animal):
+
+    def hide(self):
+        if self.hidden is False:
+            self.hidden = True
+            return self.hidden
+        self.hidden = False
+        return self.hidden
```

Comment:
you can write this in one line

Reply:
return self.hidden is True if self.hidden is False else self.hidden is False
Does not pass the tests when I use the ternary operator

Reply:
try to write like this `self.hidden = not self.hidden`